test/test_manifest.json

```diff
@@ -2408,6 +2408,21 @@
       "link": true,
       "type": "eq"
     },
+    {  "id": "jbig2_huffman_1",
+       "file": "pdfs/jbig2_huffman_1.pdf",
+       "md5": "93ccd85c5686bea27d3e6c3b41921b21",
+       "lastPage": 1,
+       "rounds": 1,
```

Comment:
You need to add `"link": true,` since otherwise the file won't be downloaded, which means that testing fails.